rest-service/manager_rest/rest/resources_v3_1/blueprints.py

```diff
@@ -100,7 +100,7 @@ def put(self, blueprint_id, **kwargs):
         created_at = args.get('created_at')
         created_by = args.get('created_by')
 
-        visibility = args.get('visibility')
+        visibility = args.get('visibility', None)
```

Comment:
`None` is the default value that is returned from dict get().

Reply:
I reverted that change. Thanks